client/Components/GameBoard/Card.jsx

```diff
@@ -66,6 +66,12 @@ class InnerCard extends React.Component {
     }
 
     isAllowedMenuSource() {
+        // Explicitly disable menus on agendas when they're selectable during a
```

Comment:
We probably need a better, more generic way of doing this, but this is fine for now

Reply:
Agreed. Originally I was going to see about making menus more dynamic on the server-side for this purpose, but it wasn't 100% clear if that'd be useful for other things and the PR was already getting pretty large, so I went with the quicker fix.